bin/rbenv-each

```diff
@@ -2,36 +2,60 @@
 #
 # Summary: Execute a command for each Ruby version
 #
-# Usage: rbenv each [-v] <command> [arg1 arg2...]
+# Usage: rbenv each [options] <command> [arg1 arg2...]
```

Comment:
I wanted this to say `[-va]` instead of `options` but I realized that our option parsing doesn't support combining multiple shorthand flags together. So these docs will do for now.